src/styles/admin.css

```diff
@@ -0,0 +1,7 @@
+.request-header {
```

Comment:
sorry for the late response, we are using tailwind this year for all of our styling so could you refactor these into tailwind classnames?

Reply:
Hey Ryan, sorry about that, I just refactored the admin component. Here is how it looks with the new changes (will commit and request review now):
<img width="400" alt="Screenshot 2024-02-10 at 3 52 52 PM" src="https://github.com/hack4impact-calpoly/ecologistics-portal/assets/70654142/62efd16e-087b-420e-8b99-15ec5f5abf7a">